### PR TITLE
Add smp_spec twitter

### LINKED ISSUE #1563: [NG] Implementation of Layer5 in the news section on homepage
**Description**
Currently Layer5 in the news section is not implemented.

**Desired Situation**

 This section should be completed and appear similar in style to [this mockup](https://www.figma.com/file/5ZwEkSJwUPitURD59YHMEN/Layer5-Designs?node-id=3359%3A0)

---
[Optional] **Screenshots**

Screen Shot 2021-02-10 at 10 15 50 PM
[Optional] **Additional context**

---
**Contributor Resources**
The `layer5` repo contains two websites. The current generation and the next-generation of the layer5.io site.

If the `layer5-ng` label is absent on this issue, then this issue pertains to the current generation of the layer5.io website, which uses Jekyll and GitHub Pages. Site content is found under the [`master` branch](https://github.com/layer5io/layer5/tree/master).
- See [`layer5` contributing instructions](https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md)

If the `layer5-ng` label is present on this issue, then this issue pertains to the next-generation of the layer5.io website, which uses Gatsby, Strapi, and GitHub Pages. Site content is found under the [`layer5-ng` branch](https://github.com/layer5io/layer5/tree/layer5-ng).
- See [`layer5-ng` contributing instructions](https://github.com/layer5io/layer5/blob/layer5-ng/CONTRIBUTING.md)

### LINKED ISSUE #1571: [README] Add smp_spec twitter handle to list
**Current Behavior**
"Find us on Twitter: @layer5 and @mesheryio. Subscribe on Youtube" statement at the bottom of the README is missing the twitter.com/smp_spec handle.

---
**Contributor Resources**

The layer5.io website uses Gatsby, Strapi, and GitHub Pages. Site content is found under the [`layer5-ng` branch](https://github.com/layer5io/layer5/tree/layer5-ng).
- See [`layer5-ng` contributing instructions](https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md)

### PR DESCRIPTION
**Pull request by Vi1234sh12**
Signed-off-by: Vi1234sh12 <vishald41234@gmail.com>

Description

This PR fixes #1571 

Notes for Reviewers

Signed commits

 - [x] Yes, I signed my commits.

### CODE REVIEW

**Comment by Nikhil-Ladha** [comment]
Can you resolve the conflict @Vi1234sh12 ? Then we are good to go.

**Comment by Vi1234sh12** [comment]
> Can you resolve the conflict @Vi1234sh12 ? Then we are good to go.

Actually, I don't know how to resolve this conflict?

**Comment by Nikhil-Ladha** [comment]
Looks good, just update the fix the image path.

**Comment by Nikhil-Ladha** [comment]
> Actually, I don't know how to resolve this conflict?

And here we come, to support you. Ping for help in slack and someone will help you out, then guess what? 
You will be rocking 🎸

File: README.md [hunk 1]
```diff
@@ -102,11 +102,10 @@ Jump into our <a href="http://slack.layer5.io"><img src=".github/assets/images/s
 <p>
 <i>Not sure where to start?</i> Grab an open issue with the <a href="https://github.com/issues?utf8=✓&q=is%3Aopen+is%3Aissue+archived%3Afalse+org%3Alayer5io+label%3A%22help+wanted%22+">help-wanted label</a>
 </p>
-<p>Find us on Twitter: <a href="https://twitter.com/layer5"><img src=".github/assets/images/twitter.png" height="16px"
-align="bottom" />@layer5</a> and <a href="https://twitter.com/mesheryio">@mesheryio</a>. Subscribe on <a href="https://www.youtube.com/channel/UCFL1af7_wdnhHXL1InzaMvA?sub_confirmation=1">Youtube</a> 
+<p>Find us on Twitter: <a href="https://twitter.com/layer5"><img src="assets/images/twitter.png" height="16px"
```
Comment (Nikhil-Ladha) [inline README.md:105]: There's a `.github` missing in the path. Check the previous code.

**Review by Nikhil-Ladha — APPROVED** [review]

**Comment by Nikhil-Ladha** [comment]
Yeaa, finally 🎉

**Comment by Vi1234sh12** [comment]
> Yeaa, finally 🎉

Thank you for helping me @Nikhil-Ladha

**Pull request merged**